crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -109,9 +112,59 @@ impl JsonRpcHandler {
     /// starknet_getTransactionByHash
     pub(crate) async fn get_transaction_by_hash(
         &self,
-        _transaction_hash: TransactionHashHex,
+        transaction_hash: TransactionHashHex,
     ) -> RpcResult<TransactionWithType> {
-        Err(error::ApiError::TransactionNotFound)
+        let starknet = self.api.starknet.read().await;
+        let transaction_to_map = starknet.transactions.get(&transaction_hash.0).unwrap();
```

Comment:
unwraps will panic on None or Err variant

Reply:
Fixed in 92e13839994735e8a2937c7e04da7e2cad36a437